go.mod

```diff
@@ -0,0 +1,129 @@
+module github.com/operator-framework/operator-sdk
+
+go 1.12
+
+require (
+	github.com/BurntSushi/toml v0.3.1
+	github.com/coreos/go-semver v0.2.0
+	github.com/coreos/prometheus-operator v0.29.0
+	github.com/ghodss/yaml v1.0.0
+	github.com/go-logr/logr v0.1.0
+	github.com/go-logr/zapr v0.1.1
+	github.com/iancoleman/strcase v0.0.0-20180726023541-3605ed457bf7
+	github.com/markbates/inflect v1.0.4
+	github.com/martinlindhe/base36 v0.0.0-20180729042928-5cda0030da17
+	github.com/mattbaird/jsonpatch v0.0.0-20171005235357-81af80346b1a
+	github.com/mitchellh/go-homedir v1.1.0
+	github.com/operator-framework/operator-lifecycle-manager v0.0.0-20190128024246-5eb7ae5bdb7a
+	github.com/pborman/uuid v1.2.0
+	github.com/pkg/errors v0.8.1
+	github.com/prometheus/client_golang v0.9.3-0.20190127221311-3c4408c8b829
+	github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90
+	github.com/prometheus/common v0.2.0
+	github.com/prometheus/prometheus v0.0.0-20190328180107-4d60eb36dcbe
+	github.com/rogpeppe/go-internal v1.3.0
+	github.com/sergi/go-diff v1.0.0
+	github.com/sirupsen/logrus v1.4.1
+	github.com/spf13/afero v1.2.2
+	github.com/spf13/cobra v0.0.3
+	github.com/spf13/pflag v1.0.3
+	github.com/spf13/viper v1.3.2
+	github.com/stretchr/testify v1.3.0
+	go.uber.org/zap v1.9.1
+	golang.org/x/tools v0.0.0-20190408170212-12dd9f86f350
+	gopkg.in/yaml.v2 v2.2.2
+	k8s.io/api v0.0.0-20190612125737-db0771252981
+	k8s.io/apiextensions-apiserver v0.0.0-20190228180357-d002e88f6236
+	k8s.io/apimachinery v0.0.0-20190612125636-6a5db36e93ad
+	k8s.io/cli-runtime v0.0.0-20181213153952-835b10687cb6
+	k8s.io/client-go v11.0.0+incompatible
+	k8s.io/code-generator v0.0.0-20181203235156-f8cba74510f3
+	k8s.io/gengo v0.0.0-20190327210449-e17681d19d3a
+	k8s.io/helm v2.13.1+incompatible
+	k8s.io/klog v0.3.1
+	k8s.io/kube-openapi v0.0.0-20190320154901-5e45bb682580
+	k8s.io/kube-state-metrics v1.6.0
+	k8s.io/kubernetes v1.11.8-beta.0.0.20190124204751-3a10094374f2
+	sigs.k8s.io/controller-runtime v0.1.10
+	sigs.k8s.io/controller-tools v0.0.0-20190411181648-9d55346c2bde
+)
+
+replace (
+	k8s.io/api => k8s.io/api v0.0.0-20181213150558-05914d821849
```

Comment:
Can we add a comment here for which k8s version tags these dependencies are pinned to?
Like we do in the go.mod scaffold for the projects.
https://github.com/operator-framework/operator-sdk/blob/master/internal/pkg/scaffold/go_mod.go#L77

It's hard to tell from the psuedo-versions what k8s versions we're using.